db/models.py

```diff
@@ -14,3 +16,35 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor)
+    genres = models.ManyToManyField(Genre)
+
+    def __str__(self) -> str:
+        return f"{self.title}, {self.description}"
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    @property
+    def capacity(self):
+        return self.rows * self.seats_in_row
+
+    def __str__(self) -> str:
+        return f"{self.name}, {self.rows}, {self.seats_in_row}"
+
+
+class MovieSession(models.Model):
+    show_time = models.DateTimeField(null=True)
+    cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        return f"{self.movie.title} {self.show_time.strftime("%Y-%m-%d %H:%M:%S")}"
```

Comment:
There is a syntax error in the `strftime` method call. The double quotes inside the string should be escaped or changed to single quotes to avoid syntax issues.